src/Configuration/Config.NewtonsoftJson/test/Microsoft.Extensions.Configuration.NewtonsoftJson.Tests.csproj

```diff
@@ -5,12 +5,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Config\test\Microsoft.Extensions.Configuration.Tests.csproj" />
-    <ProjectReference Include="..\..\testassets\Test.Common\Microsoft.Extensions.Configuration.Test.Common.csproj" />
+    <Compile Include="..\..\Config\test\ConfigurationProviderTestBase.cs">
+      <Link>test\ConfigurationProviderTestBase.cs</Link>
+    </Compile>
+    <Compile Include="..\..\testassets\Test.Common\ConfigurationProviderExtensions.cs">
+      <Link>test\Common\ConfigurationProviderExtensions.cs</Link>
+    </Compile>
+    <Compile Include="..\..\testassets\Test.Common\TestStreamHelpers.cs">
```

Comment:
If the project needs to be deleted, move these files out of testassets/ and into something that's more clearly shared source

Reply:
OK moved them next to the other Common file.